stdlib/tests/collections/mmr.rs

```diff
@@ -213,3 +213,140 @@ fn test_mmr_tree_with_one_element() -> Result<(), MerkleError> {
 
     Ok(())
 }
+
+#[test]
+fn test_mmr_unhash() {
+    let number_of_leaves = 0b10101; // 3 peaks, 21 leaves
```

Comment:
It would also be great to test the case with more than 16 peaks to make sure that the loop part is working as expected.

Reply:
added